src/operator/pairwise.ts

```diff
@@ -8,12 +8,13 @@ import {Subscriber} from '../Subscriber';
  * The (N-1)th is stored in the internal state until Nth input occurs.
  * @returns {Observable<R>} an observable of pairs of values.
  */
-export function pairwise<T>(): Observable<T> {
-  return this.lift(new PairwiseOperator());
+export function pairwise<T>(): Observable<[T, T]> {
+  let _this: Observable<T> = this;
```

Comment:
for naming specific I also vote for self, in addition to @david-driscoll 's suggestion
- it's name already using in codebases
- generally in RxJS codebase avoid underbar

for casting itself, let me follow context bit more..